lib/tss/response_handlers.rb

```diff
@@ -16,4 +16,12 @@ def handle_get_response(response, path, root_name)
     else fail "TSS: 'GET #{path}' failed with status = #{response.code}."
     end
   end
+
+  def handle_put_response(response, path)
+    case response.code
+    when 200 then JSON.parse(response.body)
+    when 404 then nil
+    else fail "TSS: 'PUT #{path}' failed with status = #{response.code}."
+    end
+  end
```

Comment:
Thanks, it looks good, I'll try to use this one (we also have handle the nil response in case of 404 error according to the specs)